src/Compilers/VisualBasic/Portable/Errors/Errors.vb

```diff
@@ -1957,7 +1957,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         WRN_Experimental = 42380
 
         ' // AVAILABLE                             42381 - 49998
-        ERRWRN_Last = WRN_Experimental
+        ERRWRN_NextAvailable = 42381
```

Comment:
Members of a helper class would still need to be updated as `ERRID` changes though.